Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -5242,7 +5242,13 @@ void Document::update_animations_and_send_events(Optional<double> const& timesta
     //   updated.
     // - Queueing animation events for any such animations.
     m_last_animation_frame_timestamp = timestamp;
+
+    GC::RootVector<GC::Ref<Animations::AnimationTimeline>> timelines_to_update(heap());
+    timelines_to_update.ensure_capacity(m_associated_animation_timelines.size());
     for (auto const& timeline : m_associated_animation_timelines)
+        timelines_to_update.append(timeline);
```

Comment:
Does the copy constructor not suffice?

Reply:
Same below

Reply:
That would give us a vector of GC::Ref that nobody is responsible for visiting